tracing-subscriber/src/fmt/time/time_crate.rs

```diff
@@ -291,6 +310,128 @@ where
     }
 }
 
+// === impl OffsetTime ===
+
+#[cfg(feature = "local-time")]
+impl OffsetTime<well_known::Rfc3339> {
+    /// Returns a formatter that formats the current time using the [local time offset] in the [RFC
+    /// 3339] format (a subset of the [ISO 8601] timestamp format).
+    ///
+    /// Returns an error if the local time offset cannot be determined. This typically occurs in
+    /// multithreaded programs. To avoid this problem, initialize `OffsetTime` before forking
+    /// threads. When using Tokio, this means initializing `OffsetTime` before the Tokio runtime.
```

Comment:
it might be worth adding an example showing initializing the formatter before starting a runtime? not a blocker though.

Reply:
Done (I didn't realize I could use Tokio and write my own `main` in examples like these...).